cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/metrics.rst

```diff
@@ -138,7 +138,7 @@ The general form of a metrics request is::
 
 The scope must be either ``system`` for system metrics or ``user`` for user-defined metrics.
 
-System metrics are either Application metrics (about Applications and their Flows, Procedures, MapReduce and Workflows) or they are Data metrics (relating to Streams or Datasets).
+System metrics are either Application metrics (about Applications and their Flows, Workers, Procedures, MapReduce and Workflows) or they are Data metrics (relating to Streams or Datasets).
```

Comment:
Can we add Workers in the same location each time, after Workflows? Or is there a particular place in that list it seems  more natural to place it?